bugsnag-plugin-android-exitinfo/src/main/java/com/bugsnag/android/TombstoneParser.kt

```diff
@@ -8,6 +8,16 @@ import com.bugsnag.android.repackaged.server.os.TombstoneProtos
 import com.bugsnag.android.repackaged.server.os.TombstoneProtos.Tombstone
 import com.bugsnag.android.Thread as BugsnagThread
 
+/**
+ * from android.util.Log
+ */
+const val VERBOSE = 2
+const val DEBUG = 3
+const val INFO = 4
+const val WARN = 5
+const val ERROR = 6
+const val ASSERT = 7
```

Comment:
These should be `private` and rather added to the `TombstoneParser` `companion object`